assets/SIL.Chorus.Mercurial.targets

```diff
@@ -1,10 +1,9 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
 		<RuntimesDirectory>$(MSBuildThisFileDirectory)\..\runtimes</RuntimesDirectory>
-	</PropertyGroup>
-
-	<!-- Make CopyFiles target actually get executed.-->
-	<PropertyGroup>
+		<!-- Allow extraction to a custom location -->
+		<Mercurial4ChorusParentDir Condition="'$(Mercurial4ChorusParentDir)'==''">$(SolutionDir)</Mercurial4ChorusParentDir>
```

Comment:
I wonder if it would be better to name it `Mercurial4ChorusExtractionDir`?

Reply:
It'll still be extracted into subdirectories Mercurial and MercurialExtensions, but that sounds reasonable.